docs/topics_5/web-openshift-deploy.adoc

```diff
@@ -1,26 +1,24 @@
 [[deploy_mta_app_openshift]]
-= Deploy the {ProductShortName} Application
+= Deploying the {ProductShortName} application
 
 . Download the {WebName} from the link:https://developers.redhat.com/products/mta/download[{ProductShortName} Download page].
 . Extract the ZIP file to a directory of your choice.
 +
-The path to the directory created by unzipping this file is referred to as `_MTA_HOME_` throughout this guide.
+The path to this directory is referred to as `_MTA_HOME_` throughout this guide.
 . Access the OpenShift web console. It should appear similar to the following image.
 +
 image::openshift-console.png[]
 
-. Click the `Import YAML / JSON` button in the top-right corner of the web console.
-. Provide the following information in the window that appears, as seen below.
+. Click *Import YAML / JSON* in the upper-right corner of the web console.
+. In the window that opens, enter:
```

Comment:
I'm fairly certain we never use "in the window that opens". Please change to "Enter the following information:"